hostfactory/host_provider/src/cyclecloud_provider.py

```diff
@@ -868,7 +876,31 @@ def _deperecated_terminate_status(self, input_json):
                 
                 if machines_to_terminate:
                     logger.warning("Re-attempting termination of nodes %s", machines_to_terminate)
-                    self.cluster.terminate(machines_to_terminate)
+                    try:
```

Comment:
This needs a unit test, but looks correct.
The unit test will help document the expected output

Reply:
updated the unit test